src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,4 +12,13 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    public static final Prefix PREFIX_GENDER = new Prefix("g/");
```

Comment:
Cosmetic but we should just bunch them up with together with the other prefixes.